code/game/gamemodes/changeling/powers/strained_muscles.dm

```diff
@@ -36,14 +45,11 @@
 
 		stacks++
 		//user.take_organ_damage(stacks * 0.03, 0)
-		user.staminaloss += stacks * 1.3 //At first the changeling may regenerate stamina fast enough to nullify fatigue, but it will stack
-
-		if(stacks == 11) //Warning message that the stacks are getting too high
-			to_chat(user, "<span class='warning'>Our legs are really starting to hurt...</span>")
+		user.mind.changeling.chem_charges -= stacks * 3 //At first the changeling may regenerate chemicals fast enough to nullify fatigue, but it will stack
```

Comment:
I think the fact that the cost progressively increases should be reflected in the description